internal/controlplane/handlers_oauth.go

```diff
@@ -280,6 +280,22 @@ func (s *Server) processOAuthCallback(ctx context.Context, w http.ResponseWriter
 	if err != nil {
 		return fmt.Errorf("error encoding token: %w", err)
 	}
+ 	login, err := s.ghLite.GetLoginByToken(ctx, token);
+	if (err != nil) {
+		return fmt.Errorf("error getting login: %w", err)
+	}
+	// Verify if the token user is a member of the ownerFilter (Organization)
+	if (stateData.OwnerFilter.Valid) {
+		member, err := s.ghLite.IsMemberInOrganization(ctx, token, stateData.OwnerFilter.String, login)
```

Comment:
What do you think about using the [List org memberships for authenticated users](https://docs.github.com/en/rest/orgs/members?apiVersion=2022-11-28#list-organization-memberships-for-the-authenticated-user) API instead? Then we would only need 1 API call instead of 2.

Reply:
Good catch. Yes this API serves the purpose here.